MapboxARKit/AnnotationManager.swift

```diff
@@ -85,6 +90,19 @@ extension AnnotationManager: ARSCNViewDelegate {
         delegate?.session?(session, cameraDidChangeTrackingState: camera)
     }
     
+    
+    public func renderer(_ renderer: SCNSceneRenderer, updateAtTime time: TimeInterval) {
+        delegate?.MBRenderer!(renderer, time: time)
```

Comment:
This will cause a crash if the delegate is not nil but does not implement this method. This and all other calls should not force unwrap and should use optional chaining instead. `MBRenderer` should be chained to `renderer` here at the call site, too, after you make the change noted above.